Dockerfile

```
@@ -1,31 +1,19 @@
-FROM php:7.2-cli-stretch
+FROM php:7.4-cli-alpine
+ARG WS_VERSION
+ARG HELM_VERSION=2.16.7
```

```suggestion
ARG HELM_VERSION=2.17.0
```
Contains fix for new deprecated stable helm charts location